### ISSUE TITLE
Add Unified Icons as a dependency and use the icons from the dependency

### ISSUE THREAD

**Issue by zeeshanabid94** [zeeshanabid94/ZoningToolkit#4]
Unified Icons is a great library. We are using icons from it however we are copying them over for now. We should use the library as a direct dependency.

link to github for icons: https://github.com/algernon-A/UnifiedIconLibrary

**Comment by kendallroth** [comment]
Now that I think about it, we could probably just directly reference CSII icons (since the upgrade to likely won't revert go away, or probably change paths 🤷‍♂️). Images (and paths) can be found with the debugging tools in inspector, and have paths like `Media/common/???.svg`. Hard coding like that could be _potentially_ risky if they would remove or rename something; however, it feels unlikely they would ever do that for such an in-built tool. On the other hand, it reduces dependencies (particularly for a single icon), so there's that... 🤷‍♂️.

**Comment by zeeshanabid94** [comment]
Having one less dependency is a good idea.  Though, I can imagine that we might use other icons for other different features we might introduce and they may come from the Unified icons. 

However, I also like the Unified icons better than anything built in. 

Do you see icons for our features in the game?

Also, we could keep copies of the icons we use in the repo without taking the dependency on Unified icons (like in your current PR). I don't know if copying from the Unified Icons Mod is in compliance with whatever license they have. We could just give the Mod credit in the README file. 🤔

**Comment by kendallroth** [comment]
Yeah, if additional/future workflows within this mod require other icons, I could see using the Unified icons. Although as you also said, if we don't _need_ a dependency it's generally nicer to avoid having them unnecessarily. In this case though the upgrade tool  icon from Unified is almost identical to the in-game one (which is odd, actually), so we _probably_ could use the built-in one (unless we want the guaranteed stability of providing our own) 🤷.

I had checked for other icons (ie zoning-related) in Unified, but not found any as it's more related to "entities" rather than "concepts" (which zoning falls under more). That being said, future workflows/ideas might be able to benefit from Unified icons?

The Unified library uses a [Apache 2.0 license](https://choosealicense.com/licenses/apache-2.0/), which is a relatively permissive license, allowing modification and distribution (with some non-applicable limitations). However, I do like the idea of mentioning it in the `README`.

**Comment by kendallroth** [comment]
Verified that the Unified and built-in (`Media/Tools/Net **Tool/Replace.svg`) "replace" icons are effectively identical (for what it's worth). I haven't made any changes in #3 in order to both give time to figure them out as well as handle them in association with this task.

![image](https://github.com/user-attachments/assets/0e5b461e-17a3-4f63-a7c3-d86ebb9e147a)
